app/assets/stylesheets/registration.scss

```diff
@@ -36,15 +36,18 @@
 	overflow: visible;
 }
 
-.ui-controlgroup > .ui-controlgroup-item {
+.ui-controlgroup>.ui-controlgroup-item {
 	float: left;
 	margin-left: 0;
 	margin-right: 0;
+	background: #fff 50% 50% repeat-x;
```

Comment:
Why are you positioning the background here?

Reply:
Copy paste from the line below. If it's not needed, it was already there.

Reply:
do we need it for something?